pkg/controller/backend/backend_controller.go

```diff
@@ -0,0 +1,223 @@
+package backend
```

Comment:
If I've looked correctly at the code, I understand currently deletion of Backends and deletion of Products on the 3scale side when deleting the corresponding CRs is not supported right?

Reply:
Backend delete https://issues.redhat.com/browse/THREESCALE-5473
Product Delete https://issues.redhat.com/browse/THREESCALE-5475

Reply:
:+1: